draft-ietf-quic-qpack.md

```diff
@@ -523,6 +530,12 @@ blocks within a stream have been fully processed.
 ~~~~~~~~~~
 {:#fig-header-ack title="Header Acknowledgement"}
 
+An encoder MUST treat receipt of a Header Acknowledgment as also acknowledging
+any dynamic table entries that the header block referenced.  That is, this
+instruction is also processed as a Table Size Synchronize instruction with a
+value matching the Largest Reference of the corresponding header block.
```

Comment:
Not quite, because TSS is a count of the inserts received since the last TSS, instead of an absolute index.

Reply:
Damn, really?  That's more efficient, I agree, but I completely missed that.  (My fault, not the spec.)  Since these are on the same stream, I think that we can make TSS incremental and avoid having two counters.

Now I have to fix my code...

Reply:
You mean TSS will be incremental from the largest acknowledged largest reference or most recent TSS?

Reply:
Yeah.  That way you maintain a single value for largest acknowledged entry and that is updated by TSS and header acknowledgment.  It's simpler and ensures correctness more directly.